go/chat/server_test.go

```diff
@@ -3467,6 +3523,29 @@ func TestChatSrvUserReset(t *testing.T) {
 			},
 		})
 		require.NoError(t, err)
+
+		t.Logf("user 2 gets PUK and tries to do stuff")
+		require.NoError(t, users[2].Login(g2))
+		for i := 0; i < 15; i++ {
+			_, err = ctc.as(t, users[2]).chatLocalHandler().PostLocal(ctx2, chat1.PostLocalArg{
+				ConversationID: conv.Id,
+				Msg: chat1.MessagePlaintext{
+					ClientHeader: chat1.MessageClientHeader{
+						Conv:        conv.Triple,
+						MessageType: chat1.MessageType_TEXT,
+						TlfName:     conv.TlfName,
+					},
+					MessageBody: chat1.NewMessageBodyWithText(chat1.MessageText{
+						Body: "Hello",
+					}),
+				},
+			})
+			if err == nil {
+				break
+			}
+			time.Sleep(2 * time.Second)
```

Comment:
does it always take ~2seconds to work?  too bad to add such long timeouts into test.  if it sometimes gets done faster, maybe sleep a smaller amount, but back off?

Reply:
I found it rarely completed in 2 seconds, so I'm not sure we gain much here. Whole test takes like 14 seconds on my local machine.

Reply:
hm ok.  is it waiting on team rekeyd to do its thing?

Reply:
I guess? I can take a closer look.

Reply:
team_rekeyd can be accelerated if that's useful... for this sort of thing...

Reply:
Is there an example somewhere in client tests?

Reply:
`./teams_test.go:func kickTeamRekeyd(g *libkb.GlobalContext, t libkb.TestingTB) {` in systests.  it may or may not be relevant here...

Reply:
Done